packages/core/src/components/table/table/test/column-filtering/header-input-wrapper/table.e2e.ts

```diff
@@ -0,0 +1,16 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath =
+  'src/components/table/table/test/column-filtering/header-input-wrapper/index.html';
+
+test.describe.parallel('tds-table-editable-header-cells', () => {
```

Comment:
Since this tests the interactions of the column filtering input rather than the editable cells, should this description be updated to reflect that instead?